resources/suite-runner.mjs

```diff
@@ -75,7 +75,8 @@ export class SuiteRunner {
             if (this.#client?.willRunTest)
                 await this.#client.willRunTest(this.#suite, test);
 
-            const testRunner = new TestRunner(this.#frame, this.#page, this.#params, this.#suite, test, this._recordTestResults);
+            const testRunnerClass = TEST_RUNNER_LOOKUP[this.#suite.type ?? "default"];
```

Comment:
Instead of changing the test runner based on the suite type for some suites, can we initially add a toggle to developer menu so that we can try out async runner?

Reply:
To my understanding the agreement in the meeting was to unconditionally enable the new runner for the preact workloads and add the global developer-mode menu in a separate PR.

Reply:
correct - below is the list on enabled suites (for now):

-  TodoMVC-React-Complex-DOM,
-  TodoMVC-React-Redux,
-  TodoMVC-Preact-Complex-DOM, (technically not react based, but it's similar)
-  NewsSite-Next,
-  React-Stockcharts-SVG

Reply:
If we're doing that, we need before/after numbers across browsers.

Reply:
I'll run crossbench for the ones on the list and post results here.

Reply:
Here are metrics:
![Screenshot 2024-12-18 at 5 54 40 PM](https://github.com/user-attachments/assets/daf49729-dac6-4bbc-b4c3-b83a76c82bae)